spec/latest/core/core.jsonld

```diff
@@ -428,7 +434,7 @@
     {
       "@id": "hydra:IriTemplate",
       "@type": "hydra:Class",
-      "subClassOf": "hydra:Resource",
+      "subClassOf": ["hydra:Template", "hydra:Resource"],

```

Comment:
In my opinion the weirder part is actually introducing inheritance here. `IriTemplate` is not just any string template. It is meant as implementing the [rfc6570](https://tools.ietf.org/html/rfc6570). The changes here make the relation with its original RFC... complicated.

Reply:
rfc6570 is just the default:

> The literal's datatype indicates the template syntax; if not specified, hydra:Rfc6570Template is assumed.

And this quote only describes the [`hydra:template`](https://www.hydra-cg.com/spec/latest/core/#hydra:template) property, not the `hydra:IriTemplate`

Nevertheless I also have a bad feeling with this type of inheritance.

Reply:
Yep, RFC 6570 is not directly bound to IriTemplate. In RDF representation of hydra spec it is connected only via `seeAlso`, thus no strict data type is enforced.